src/Utopia/Messaging/Adapters/Email/Brevo.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\Email;
+
+use Utopia\Messaging\Messages\Email;
+use Utopia\Messaging\Adapters\Email as EmailAdapter;
+
+class Brevo extends EmailAdapter
+{
+    public function __construct(private string $apiKey) 
+    {
+    }
+
+    public function getName(): string
+    {
+        return 'Brevo';
+    }
+
+    public function getMaxMessagesPerRequest(): int
+    {
+        return 1000;
```

Comment:
Can you link the documentation to support this in the PR description or as a response to this comment?